Library/Formula/hue.rb

```diff
@@ -0,0 +1,157 @@
+class Hue < Formula
+  desc "Hue is a Web interface for analyzing data with Apache Hadoop."
+  homepage "http://gethue.com/"
+  url "https://github.com/cloudera/hue/archive/release-3.8.1.tar.gz"
+  # version "3.8.1" # redundant with version scanned from URL
+  sha256 "582777f567b9f4a34e4ce58cfd5ed24aff15f3e02f193e7990329d8cb0161a8a"
+
+  # system dependencies - either Homebrew or Mac OS versions
+  depends_on :java
+  depends_on :python
+  depends_on :mysql
+
+  depends_on "maven" => :build
+  depends_on "openssl" => :optional
+  depends_on "gmp"
+
+  patch :DATA
+
+  def install
+    # patched the Makefile vars to install to ${PREFIX}/libexec
+    system "make", "-j", "1", "install", "PREFIX=#{prefix}"
+
+    (libexec/"desktop/conf").install "desktop/conf.dist/hue.ini"
+    etc.install_symlink "#{libexec}/desktop/conf/hue.ini"
+
+    bin.install_symlink "#{libexec}/build/env/bin/hue"
+  end
+
+  test do
+    fork do
+      system "#{bin}/hue", "runserver"
+    end
+    sleep(6)
+
+    begin
+      system "curl", "-s", "-S", "-f", "-L", "http://localhost:8000"
+      assert_equal 0, $?.exitstatus
+    ensure
+      sleep(2)
+      system "pkill", "-f", "hue"
+    end
+  end
+end
+__END__
+diff --git a/desktop/core/ext-py/tablib-develop/setup.py b/desktop/core/ext-py/tablib-develop/setup.py
+index fb16546..6f3bcfe 100755
+--- a/desktop/core/ext-py/tablib-develop/setup.py
++++ b/desktop/core/ext-py/tablib-develop/setup.py
```

Comment:
Would Hue need this or a rebase on latest tablib? (that way you don't need this here)